dotCMS/src/main/java/com/dotcms/analytics/metrics/Parameters.java

```diff
@@ -5,15 +5,9 @@
 public abstract class Parameters {
```

Comment:
This could be an interface

Reply:
Really it does not look like a interface I turned it into a final class https://github.com/dotCMS/core/pull/26586/commits/3b03fe7de9c28ae9bf5fbd50d743fa7917b39f10#diff-110bd545d2cc04c0677a40dbf6f135e072028d5ea7502ffa6ec3f98a7a5be81dR5